README.md

```diff
@@ -285,6 +293,26 @@ require("scissors").setup {
 	snippetSelection = {
 		picker = "auto", ---@type "auto"|"telescope"|"snacks"|"vim.ui.select"
 
+		fzfLua = {
+			-- same format as fzf_opts in `:h fzf-lua-customization`
+			fzf_opts = {},
+
+			-- suppress warnings from fzf-lua.
+			-- This is true by default, since commonly-used fzf-lua presets
+			-- create warnings due to border settings.
+			silent = true,
+
+			-- same format as winopts in `:h fzf-lua-customization`
+			--- @type fzf-lua.config.Winopts
+			winopts = {
+				preview = {
+					hidden = false,
```

Comment:
Technically, fzf-lua allows hiding the preview window by default. I thought this might negate a lot of the benefit to having an integration in the first place, so I set an override in the default config.

There is something to be said for just using whatever the user configured before, though... if you'd rather we respect that a user has told fzf not to show previews by default, I'm happy to remove this.